src/main/flight/mixer.c

```diff
@@ -732,6 +732,13 @@ void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
 
     // Calculate voltage compensation
     const float vbatCompensationFactor = vbatPidCompensation ? calculateVbatPidCompensation() : 1.0f;
+    
+    // Apply battery voltage compensation to throttle
+    if(currentPidProfile->vbatThrottleCompensationEnabled) {     
+        float vbatThrottleCompensationFactor = 
+            constrainf( (float)currentPidProfile->vbatThrottleCompensationVoltage / getBatteryAverageCellVoltage(), 0.5f, 1.5f );
```

Comment:
I think all we need to change is the low end of the constraint range to be 1.0f insetad of 0.5f.  That still allows a start point but eliminates the reduction above the threshold.